service/src/test/java/bio/terra/workspace/service/workspace/flight/CreateGoogleContextFlightTest.java

```diff
@@ -148,7 +148,8 @@ private static FlightMap createInputParameters(
    * CustomGcpIamRoleMapping}
    */
   private void assertRolesExist(Project project) throws IOException {
-    for (CustomGcpIamRole customRole : CustomGcpIamRoleMapping.CUSTOM_GCP_IAM_ROLES.values()) {
+    for (CustomGcpIamRole customRole :
+        CustomGcpIamRoleMapping.CUSTOM_GCP_RESOURCE_IAM_ROLES.values()) {
       String fullRoleName =
           "projects/" + project.getProjectId() + "/roles/" + customRole.getRoleName();
```

Comment:
make helper methods to build the FQ names for roles and projects

Reply:
turns out there's already a helper CustomGcpIamRole. nice catch!